README.md

```diff
@@ -50,16 +52,17 @@ This will produce files inside `lib/generated` directory.
 
 #### Upload main ARB file
 
-      flutter pub run intl_utils:localizely_upload_main --project-id <PROJECT_ID> --api-token <API_TOKEN>
+      flutter pub run intl_utils:localizely_upload_main --project-id <PROJECT_ID> --api-token <API_TOKEN> [--arb-path <ARB_PATH>]
```

Comment:
I'm just thinking... 🤔

Why do you need an additional argument for the ARB files directory?
The current implementation is not perfect, but we should have a clear idea of why we are doing some things (we can't easily change config later).

Note that we are currently reading some configuration for uploading and downloading from the `PubspecConfig` before making API calls.

Reply:
My understanding is, that if you change the directory where your arb files are, the synchronization with your platform does not work anymore, if you do not also pass the current directory as an argument. I did not manually test it though. I just added these line to pass your automatic tests and to fix all errors resulting from the new custom path parameter.